jwst/assign_wcs/nirspec.py

```diff
@@ -509,14 +537,30 @@ def get_open_msa_slits(msa_file, msa_metadata_id, dither_position,
     msa_data = [x for x in msa_conf.data if x['msa_metadata_id'] == msa_metadata_id \
                 and x['dither_point_index'] == dither_position]
 
+    # Get all source_ids for slitlets with sources.
+    # These should not be used when assigning source_id to background slitlets.
+    source_ids = set([x[5] for x in msa_conf.data if x['msa_metadata_id'] == msa_metadata_id \
+                      and x['dither_point_index'] == dither_position])
+    # All BKG shutters get a source_id value of 0.
```

Comment:
By "BKG shutters" do you mean the individual shutters of a larger slit that don't happen to have the source located in them in this particular exposure (e.g. the top and bottom shutters of a 1x3 slit when the source is in the middle shutter)? As opposed to shutters that explicitly designated as background only.

Reply:
I meant that's what BKG shutters are assigned in the msa meta file. I'll clarify the comment.